src/App.tsx

```diff
@@ -1,61 +1,131 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+const allowedCharacters
+  = 'abcdefghijkmnopqrstuvwxyzабвгґдеєжзиіїйклмнопрстуфхцчшщьюя123456789 ';
```

Comment:
Do you really need to hardcore all the possible chars?
Try to find another solution